TESTS/mbedmicro-rtos-mbed/malloc/main.cpp

```diff
@@ -7,7 +7,11 @@
 #endif
 
 #define NUM_THREADS         5
+#if defined(__CORTEX_A9)
+#define THREAD_STACK_SIZE   DEFAULT_STACK_SIZE
```

Comment:
my question would be - whats the difference between ``DEFAULT_STACK_SIZE `` and 256 ?